config/testgrids/kubernetes/sig-network/config.yaml

```diff
@@ -28,10 +28,10 @@ dashboards:
       test_group_name: ci-kubernetes-e2e-gce-cos-k8sstable1-ingress
       alert_options:
         alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
-    - name: gce-cos-1.14-ingress
-      test_group_name: ci-kubernetes-e2e-gce-cos-k8sbeta-ingress
-      alert_options:
-        alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+    # - name: gce-cos-1.14-ingress
+    #   test_group_name: ci-kubernetes-e2e-gce-cos-k8sbeta-ingress
+    #   alert_options:
+    #     alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
```

Comment:
We'll need to uncomment this after reintroducing k8s-beta, I'll leave a TODO for that later.